src/components/Profile/ProposalsCreatedBox.tsx

```diff
@@ -1,55 +1,47 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
 import { Container } from 'decentraland-ui/dist/components/Container/Container'
 
-import usePaginatedProposals from '../../hooks/usePaginatedProposals'
-import Empty from '../Common/Empty'
-import FullWidthButton from '../Common/FullWidthButton'
-import SkeletonBars from '../Common/SkeletonBars'
-import Watermelon from '../Icon/Watermelon'
+import BoxTabs from '../Common/BoxTabs'
+import BoxTabsContainer from '../Common/BoxTabsContainer'
+import BoxTabsContentContainer from '../Common/BoxTabsContentContainer'
 
-import { ProfileBox } from './ProfileBox'
-import ProposalCreatedItem from './ProposalCreatedItem'
+import CoAuthoringTab from './CoAuthoringTab'
 import './ProposalsCreatedBox.css'
+import ProposalsCreatedTab from './ProposalsCreatedTab'
 
 interface Props {
   address: string
 }
 
-const PROPOSALS_PER_PAGE = 5
+enum Tab {
```

Comment:
We have a similar enum defined in ActivityBox.tsx, does it make sense to merge them?

```typescript
enum Tab {
  MyProposals = 'myProposals',
  Watchlist = 'watchlist',
  CoAuthoring = 'coauthoring',
}
```

Reply:
Makes sense. Maybe MyProposals can be ProposalsCreated for both. Where can we put this frontend-only enum?

Reply:
Good question. 
How about one of the two components that use this? 
Or the profile page?
I don't think we have a frontend only constants file. We have `src/constants.ts`

Reply:
Let me check something... I think I can unify both tab components (ActivityBox + ProposalsCreatedBox) and that solves this issue.